weblate/utils/tasks.py

```diff
@@ -82,16 +80,22 @@ def database_backup():
         cmd += ["--port", database["PORT"]]
     if database["USER"]:
         cmd += ["--username", database["USER"]]
+    if settings.BACKUP_COMPRESSED:
+        cmd += ["--file", data_dir("backups", "database.sql.gz")]
+        cmd += ["--format", "custom"]
```

Comment:
Naming the file `sql.gz` with custom format is confusing.

Reply:
> Naming the file sql.gz with custom format is confusing.

Hmm I guess I used the wrong set of options, should have used plain format with compress options since, according to the doc:
> For plain text output, setting a nonzero compression level causes the entire output file to be compressed, as though it had been fed through gzip

Or do you suggest something else?